slatedb/src/db_state.rs

```diff
@@ -322,7 +321,9 @@ pub(crate) struct CoreDbState {
     pub(crate) l0_last_compacted: Option<Ulid>,
     pub(crate) l0: VecDeque<SsTableHandle>,
     pub(crate) compacted: Vec<SortedRun>,
-    pub(crate) next_wal_sst_id: u64,
+    /// the last WAL ID seen when the manifest is flushed (when memtable is flushed to
+    /// L0). this WAL file might has not been flushed to disk yet.
+    pub(crate) last_seen_wal_id: u64,
```

Comment:
Nit: for consistency?

```suggestion
    pub(crate) wal_id_last_seen: u64,
```